docs/document/content/reference/distsql/syntax/rdl/rule-definition/sharding/create-sharding-binding-table-rule.cn.md

```diff
@@ -23,6 +26,7 @@ tableName ::=
 - 一个分片表只能具有一个绑定关系
 - 添加绑定关系的分片表需要使用相同的资源，并且分片节点个数相同。例如 `ds_${0..1}.t_order_${0..1}` 与 `ds_${0..1}.t_order_item_${0..1}`
 - 添加绑定关系的分片表需要对分片键使用相同的分片算法。例如 `t_order_${order_id % 2}` 与 `t_order_item_${order_item_id % 2}` 
+- 只能存在一个绑定规则，但可包含多个绑定关系
```

Comment:
so can not execute `CREATE` more than one time, please use `ALTER` to modify binding table rule.